proposals/Addresses/devnet.json

```diff
@@ -0,0 +1,158 @@
+        "chainId": 421614,
+        "isContract": false
+    },
+    {
+        "name": "AUTOGRAPH_MINTER_PAYMENT_RECIPIENT",
+        "addr": "0x0e6aCa776b3d12dd85363Fd6050eE05A4a242be9",
+        "chainId": 421614,
+        "isContract": false
+    },
+    {
+        "name": "GUARDIAN_MULTISIG",
+        "addr": "0xc3c1D74048115316dCbd66baeDA8214D2CA83C35",
+        "chainId": 421614,
+        "isContract": false
+    },
+    {
+        "name": "WETH",
+        "addr": "0xc556bae1e86b2ae9c22ea5e036b07e55e7596074",
+        "chainId": 421614,
+        "isContract": true
+    },
+    {
+        "name": "TOKEN",
+        "addr": "0x5422a3De80BA3891d663fa4EC7506A7f263c1Fd9",
+        "chainId": 421614,
+        "isContract": true
+    },
+    {
+        "name": "CORE",
+        "addr": "0x68D6B4af6668A62Fc1B21ABF3DbfA366DD1d8eC7",
+        "chainId": 421614,
+        "isContract": true
+    },
+    {
+        "name": "GLOBAL_REENTRANCY_LOCK",
+        "addr": "0x87D7b991540747522404c86b281E4880Cd6dE7f2",
+        "chainId": 421614,
+        "isContract": true
+    },
+    {
+        "name": "ERC1155_MAX_SUPPLY_MINTABLE_WEARABLES",
+        "addr": "0x27564B8cf86aba79b398A39B75898fe8AFf30627",
+        "chainId": 421614,
+        "isContract": true
+    },
+    {
+        "name": "ERC1155_MAX_SUPPLY_ADMIN_MINTER",
+        "addr": "0x34c775910e5CbB1511eF00Ea51cd0f6bd1E3E4Db",
+        "chainId": 421614,
+        "isContract": true
+    },
+    {
+        "name": "ADMIN_TIMELOCK_CONTROLLER",
+        "addr": "0x4a40ca2184ec0c5fab58b45b4a3a2bc240eb1043",
+        "chainId": 421614,
+        "isContract": true
+    },
+    {
+        "name": "ERC1155_MAX_SUPPLY_MINTABLE_CONSUMABLES",
+        "addr": "0x898C5e72Cb4121A8ae579faEAe2C5879196493fc",
+        "chainId": 421614,
+        "isContract": true
+    },
+    {
+        "name": "ERC1155_MAX_SUPPLY_MINTABLE_PLACEABLES",
+        "addr": "0xa7654c44f0A4Da52a10469dC380f02F75d6E0631",
+        "chainId": 421614,
+        "isContract": true
+    },
+    {
+        "name": "ERC1155_AUTO_GRAPH_MINTER",
+        "addr": "0x34c775910e5CbB1511eF00Ea51cd0f6bd1E3E4Db",
+        "chainId": 421614,
+        "isContract": true
+    },
+    {
+        "name": "GAME_CONSUMABLE",
+        "addr": "0xf052f3F94f6E71DfBA39544b8DF02c873De4469F",
+        "chainId": 421614,
+        "isContract": true
+    },
+    {
+        "name": "SEASONS_TOKEN_ID_REGISTRY",
+        "addr": "0x8ccE22Fe7Bd3F4998E158De3c3a77ee85A6F6bBB",
+        "chainId": 421614,
+        "isContract": true
+    },
+    {
+        "name": "ERC1155_SEASON_ONE",
+        "addr": "0x3006b30E32dEA503503Bf9e3f909163834085A5C",
+        "chainId": 421614,
+        "isContract": true
+    },
+    {
+        "name": "DEPLOYER",
+        "addr": "0x5Ec41e3a9c712D0BBC26d2CbA0E653c5d2cc982C",
+        "chainId": 421614,
+        "isContract": false
+    },
+    {
+        "name": "AFTER_DEPLOY_CALLER",
```

Comment:
This is no longer needed after this PR https://github.com/ZTX-Foundation/tuxedo/pull/233.

Reply:
Thanks @prateek105. Fixed.